datastore/providers/__init__.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
```

Comment:
would you mind removing this? I think it was cleaner before

Reply:
Removing which part? The import or the entire enum approach? I added this so we'd have a concrete way to define support vector dbs. There was an issue about have some clarity in the logs that would help user pick a proper datastore value. We could always hardcode it, but figure that was more prone to bugs! Happy to remove though!

Reply:
just the enum part :)